test/threads_test.py

```diff
@@ -25,14 +25,31 @@ def f2(x):
         self.assertEqual(fr.result, 2)
         self.assertEqual(fr2.result, 3)
 
-    def todo_test_do(self):
+    def test_do(self):
 
         # __doc__ (as of 2008-06-28) for pygame.threads.WorkerQueue.do:
 
-        # puts a function on a queue for running later.
+        # puts a function on a queue for running _later_.
         #
+        def sleepTest():
```

Comment:
shouldn't use camelCase style naming for function names or pylint will cry real tears.

Reply:
Good point. Revisiting this I also realized that this sleep function could be shortened without harming the test